soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.it81202bx

```diff
@@ -9,4 +9,7 @@ config SOC
 config SOC_IT8XXX2_GPIO_GROUP_K_L_DEFAULT_PULL_DOWN
 	default y
 
+config SOC_USBPD_ITE_PHY_PORT_COUNT
+	default 2
```

Comment:
Can this be in device tree? It seems it's used to describe the hardware layout, not software configuration.

Reply:
Done

Reply:
Done